src/app/reducers/group-type.reducer.spec.ts

```diff
@@ -0,0 +1,81 @@
+import * as GroupTypeActions from './../actions/group-type.actions';
+import { GroupTypeUI } from './../models/group-types.model';
+import { initialState as GroupTypeIntialState } from './../states/grouptype.state';
+import { GroupTypeReducer } from './group-type.reducer';
+
+export type Action = GroupTypeActions.All;
+
+describe('GroupTypeReducer: ', () => {
+  let groupTypes: GroupTypeUI[];
+  beforeEach(() => {
+    groupTypes = [
+      {
+        id: '679a563c-ac9b-4478-9f3e-4187f708dd30',
+        name: 'Scenarios',
+        bucket: 'portfolio',
+        level: [],
+        icon: 'fa fa-bullseye',
+        group: null,
+        selected: false,
+        showInSideBar: true,
+        typeList: null,
+        infotip: 'Lorem ipsum dolor sit amet'
+      },
+      {
+        id: '44795662-db7a-44f7-a4e7-c6d41d3eff27',
+        name: 'Requirements',
+        bucket: 'requirement',
+        level: [],
+        icon: 'fa fa-list-ul',
+        group: null,
+        selected: false,
+        showInSideBar: true,
+        typeList: null,
+        infotip: 'Lorem ipsum dolor sit amet'
+      }
+    ];
+  });
+
+  it('undefined action should return the default state', () => {
```

Comment:
Please add the class name as a prefix to the test message following by the function name if it's there.
Like this `GroupTypeReducer :: `

Reply:
I see it's there on desc. My bad !